modules/logging-common-terms.adoc

```diff
@@ -0,0 +1,84 @@
+
+Elasticsearch Operator::
+Elasticsearch operator is used to run Elasticsearch cluster on top of {product-title}. The Elasticsearch Operator provides self-service for the Elasticsearch cluster operations and is used by {product-title} Logging.
+
+indexing::
+Indexing is a data structure technique which is used to quickly locate and access data. Indexing optimizes the performance by minimizing the amount of disk access required when a query is processed.
+
+JSON logging::
+{product-title} Logging Log Forwarding API enables you to parse JSON logs into a structured object and forward them to either {product-title} Logging-managed Elasticsearch or any other third-party system supported by the Log Forwarding API.
+
+Kibana::
+Kibana is a browser-based console interface to query, discover, and visualize your Elasticsearch data through histograms, line graphs, and pie charts.
+
+Kubernetes apiserver::
+Kubernetes apiserver Validates and configures data for the API objects.
+
+Labels::
+Labels are key-value pairs that you can use to organize and select subsets of objects such as a pod.
+
+Logging::
+With {product-title} Logging you can aggregate application, infrastructure and audit logs throughout your cluster, store them to a default log store, forward them to third party systems and query, visualize the stored logs in the default log store.
+
+logging collector::
+A logging collector collects logs from the cluster, formats them, and forwards them to the log store or third party systems.
+
+log store::
+A log store is used to store aggregated logs. You can use the default Elasticsearch log store or forward logs to external log stores. The default log store is optimized and tested for short-term storage.
+
+log visualizer::
+Log visualizer is the user interface (UI) component you can use to view logs, graphs, charts, and so forth. The current implementation is Kibana.
+
+node::
+A node is a worker machine in the {product-title} cluster. A node is either a virtual machine (VM) or a physical machine.node selector
+
+Operators::
+Operators are the preferred method of packaging, deploying, and managing a Kubernetes application in an {product-title} cluster. An Operator takes human operational knowledge and encodes it into software that is packaged and shared with customers.
+
+pod::
+A pod is the smallest logical unit in Kubernetes. A pod is comprised of one or more containers to run in a worker node..
+
+Role-based access control (RBAC)::
+RBAC is a key security control to ensure that cluster users and workloads have only access to resources required to execute their roles.
+
+shards::
+Elasticsearch organizes the log data from Fluentd into datastores, or indices, then subdivides each index into multiple pieces called shards.
+
+taint::
+Taints and tolerations ensure that pods are scheduled onto appropriate nodes. You can apply one or more taints on a node.
```

Comment:
This mentions _tolerations_ but that isn't described until later. Maybe omit _tolerations_ entirely here?

Reply:
acked